Makefile

```diff
@@ -12,8 +12,8 @@ else
        TEST_DIR=${WHAT}	
 endif
 
+export GOARCHAIUS_CONFIG_PATH=$(CURDIR)
```

Comment:
Made scope of GOARCHAIUS_CONFIG_PATH global so that it is applicable to all Makefile child processes

Reply:
Great change!